### PR TITLE
Implement deletion policy

### LINKED ISSUE #447: backend: replace oqt.ohsome.org
The OQAPI integration of the Sketch Map Tool still uses `oqt.ohsome.org`. This domain is deprecated and should be replaced by `api.quality.ohsome.org`.

### PR DESCRIPTION
**Pull request by matthiasschaub**
Todo:
- [x] delete bbox on non-consent
- [ ] ~should Bbox class have from string (from wkt function)?~
- [x] uncomment esri layer 
- [x] check if autovacuum is enabled 
- [ ] ~Fix tests by setup database on each run (not just on writing into the database)~

This PR implements two cleanup functions as well as stores more metadata in the database:
1. `cleanup_map_frames`: Cleanup of old map frames for which there are uploaded sketch maps without consent. This cleanup is executed by celery beat every hour.
2. `cleanup_blobs`: Cleanup of uploaded sketch maps without consent after vector and raster results are generated. This cleanup is executed as callback by celery after uploaded sketch maps are processed.
3. Store map generation parameter in the database

Possible improvements:
- `client_celery.select_file` call in `tasks.py` should return not only the file but also `layer` and `bbox`. This would enable to load those only at the place needed and not in `routes.py` and pass it to `tasks.py`. (locality of behaviour)
- avoid code duplication in database clients
- maybe rename `select_file()` to a more specific name since it does not only return file

Closes #447

### CODE REVIEW

File: sketch_map_tool/database/client_celery.py [hunk 1]
```diff
@@ -50,13 +103,67 @@ def delete_map_frame(uuid: UUID):
...
+
+    Only set file to null. Keep metadata.
+    """
+    query = """
+    UPDATE
+        map_frame
+    SET
+        file = NULL
+    WHERE
+        ts < NOW() - INTERVAL '6 months'
```
Comment (matthiasschaub) [inline sketch_map_tool/database/client_celery.py:117]:
```suggestion
        ts < NOW() - INTERVAL '1 years'
```

File: sketch_map_tool/models.py [hunk 2]
```diff
@@ -17,6 +17,16 @@ class Bbox:
     lon_max: float
     lat_max: float
 
+    @property
```
[the remark this answers is not in the document]
Comment (Gigaszi) [reply sketch_map_tool/models.py:20]: done

File: sketch_map_tool/database/client_flask.py [hunk 3]
```diff
@@ -85,35 +88,62 @@ def set_async_result_ids(request_uuid, map_: dict[REQUEST_TYPES, str]):
...
-    """Insert uploaded files as blob into the database and return primary keys"""
+def insert_files(files, consent: bool) -> tuple[list[int], list[str], list[str]]:
+    """Insert uploaded files as blob into the database and return ID, UUID and name.
+
+    UUID is derived from decoding the qr-code.
+    """
     create_query = """
     CREATE TABLE IF NOT EXISTS blob(
         id SERIAL PRIMARY KEY,
+        uuid UUID,
```
[the remark this answers is not in the document]
Comment (Gigaszi) [reply sketch_map_tool/database/client_flask.py:99]: done

File: sketch_map_tool/database/client_celery.py [hunk 4]
```diff
@@ -50,13 +103,67 @@ def delete_map_frame(uuid: UUID):
...
+        except UndefinedTable:
+            logging.info("Table `map_frame` does not exist yet. Nothing todo.")
+
+
+def cleanup_blobs():
+    """Cleanup uploaded files (sketch maps) without consent.
+
+    Only set file and name to null. Keep metadata.
+    """
+    # TODO: Wait one day until deletion or check celery task status?
```
[the remark this answers is not in the document]
Comment (matthiasschaub) [reply sketch_map_tool/database/client_celery.py]: Implemented delete by map frame uuid.

File: sketch_map_tool/tasks.py [hunk 5]
```diff
@@ -199,8 +196,25 @@ def digitize_sketches(
...
             m: FeatureCollection = post_process(m, file_name)
             l.append(m)
     return merge(l)
+
+
+@celery.task
+def cleanup_map_frames():
+    """Cleanup map frames stored in the database."""
+    db_client_celery.cleanup_map_frames()
+    # TODO: run Vacuum or Vacuum full?
```
[the remark this answers is not in the document]
Comment (matthiasschaub) [reply sketch_map_tool/tasks.py:210]: Autovacuum should be enabled by default. Check first. If so, this should be enough.

**Review by joker234 — COMMENTED** [review]
some recommendations:
- mock arcgis request in `definitions.py`
- remove `.gitattributes`
- link config.py (defaults) in documentation

**Comment by matthiasschaub** [comment]
> some recommendations:
> 
>     * mock arcgis request in `definitions.py`
> 
>     * remove `.gitattributes`
> 
>     * link config.py (defaults) in documentation

All of those has been adressed

We adr

**Review by joker234 — APPROVED** [review]

**Pull request merged**